source/mn_menus.cpp

```diff
@@ -264,12 +263,16 @@ CONSOLE_COMMAND(mn_newgame, 0)
 
    // haleyjd 05/14/06: check for episode menu override now
    if(mn_episode_override)
-   {
       MN_StartMenu(mn_episode_override);
-      return;
+   else if(GameModeInfo->menuStartMap &&
+           *GameModeInfo->menuStartMap &&
+           W_CheckNumForName(GameModeInfo->menuStartMap) >= 0)
```

Comment:
Unsure how to format this expression given the code style I've come across so far. Is there a code fixer / linter in the project?

Really would rather it be a class predicate but... Doom source pain.

```c++
  public:
      bool hasMenuStartMap()
      {
          return menuStartMap && *menuStartMap && W_CheckNumForName(menuStartMap) >= 0;
      }
```

Reply:
That else if is in keeping with the general style of EE. Sometimes I dunno how I feel about it, but it's sticking around for now at least.